### PR TITLE
WL-4237 Removed gaps between paragraphs in Syllabus tool.

### PR DESCRIPTION
**Pull request by ouit0408**
Added margin for <p> tag in syllabus.css to remove extra space when text is
added through CKEditor.

### CODE REVIEW

**Comment by buckett** [comment]
Where was the original extra space coming from?

**Comment by buckett** [comment]
I think it's coming from this and affects more than just `<p>` tags (`<div>` etc).
screen shot 2016-02-26 at 14 41 20

**Comment by ouit0408** [comment]
it looks like in sakai syllabus tool, for textarea element deliberately 'editable-pre-wrapped ' class is added - ref SAK-23451 . But the code in jqueryui-editable.js which was using this styling has been commented out. Shall i remove this class from textarea ?

**Comment by buckett** [comment]
Just chatted about this in the office and it looks like the `editable-pre-wrapped` class is only used to preserve paragraphs and newlines. As we're editing HTML directly we don't need to worry about this. The cleanest fix is to just override the CSS as this is easier to manage in the long run, rather than trying to change the JS and merge it as newer versions of the library come out.

**Comment by ouit0408** [comment]
have commented the styling for editable-pre-wrapped class and have updated the PR.

File: syllabus-app/src/webapp/syllabus/css/jqueryui-editable.css [hunk 1]
```diff
@@ -136,7 +136,7 @@
 }
 
 .editable-pre-wrapped {
-   white-space: pre-wrap;
```
Comment (buckett) [inline syllabus-app/src/webapp/syllabus/css/jqueryui-editable.css:139]: Much better off to override this CSS iin syllabus.css as the jquery-editable.css is taken as-is from an upstream source and if a new version of the library gets pulled in this change will probably get lost.

**Comment by ouit0408** [comment]
made the change and have updated the PR

**Pull request merged**